CHANGELOG.md

```diff
@@ -150,9 +152,10 @@ be found.
 ### Swarm Mode
 
 + Add secret management [#27794](https://github.com/docker/docker/pull/27794)
++ Add support for templating service options [#28025](https://github.com/docker/docker/pull/28025)
```

Comment:
Right now it supports `hostname`, `mounts` and `environment variables` so we might wanna say *for certain options* :angel: